.github/workflows/ros2-ci.yaml

```
@@ -9,37 +9,44 @@ on:
 jobs:
 
   build-ros2:
-    runs-on: ubuntu-latest
+    runs-on: [ubuntu-latest]
     strategy:
       fail-fast: false
       matrix:
-        ros-version: [foxy, galactic]
+        ros_distribution:
+          - foxy
+          - galactic
+          - humble
+          - rolling
         include:
-          - ros-version: foxy
-            image: ros:foxy
-          - ros-version: galactic
-            image: ros:galactic
+          # Foxy Fitzroy (June 2020 - May 2023)
+          - docker_image: rostooling/setup-ros-docker:ubuntu-focal-ros-foxy-ros-base-latest
+            ros_distribution: foxy
```

[nits/IMO] I feel writing the key data first is better.

```suggestion
          - ros_distribution: foxy
            docker_image: rostooling/setup-ros-docker:ubuntu-focal-ros-foxy-ros-base-latest
```